e2e/wdio/core/tests/action-sheet.e2e-spec.ts

```diff
@@ -0,0 +1,76 @@
+import { ActionSheetPo } from '../pages/action-sheet.po';
+import {
+    click,
+    getAttributeByName,
+    getElementArrayLength,
+    getImageTagBrowserPlatform,
+    refreshPage,
+    saveElementScreenshot,
+    waitForElDisplayed,
+    checkElementScreenshot,
+    scrollIntoView
+} from '../../driver/wdio';
+import { checkElArrIsClickable } from '../../helper/assertion-helper';
+import { classAttribute, compactValue } from '../fixtures/appData/action-sheet-content';
+
+describe('Action sheet test suite', function() {
+    const actionSheetPage = new ActionSheetPo();
+    const {
+        actionSheetMenuButton,
+        actionSheetList,
+        actionSheetListItemButtons,
+        alertMessage
+    } = actionSheetPage;
+
+    beforeAll(() => {
+        actionSheetPage.open();
+    }, 1);
+
+    afterEach(() => {
+        refreshPage();
+        waitForElDisplayed(actionSheetMenuButton);
+    }, 1);
+
+    it('should check action sheet items are clickable', () => {
+        const actionSheetCount = getElementArrayLength(actionSheetMenuButton);
+
+        for (let i = 0; actionSheetCount > i; i++) {
+            click(actionSheetMenuButton, i);
+            checkElArrIsClickable(actionSheetListItemButtons);
+        }
+    });
+
+    it('should check comfy and cozy properties', () => {
```

Comment:
Maybe check the actual size or create a screenshot?

Reply:
I already take the screenshot of these action sheets in "should check action sheet items visual regression"